File: app/src/main/java/ca/warp7/android/scouting/abstraction/AbstractActionVibrator.java
package ca.warp7.android.scouting.abstraction;

/**
 * @since v0.4.2
 */

public interface AbstractActionVibrator {

    /**
     * Vibrate to indicate scouting start
     */
    void vibrateStart();

    /**
     * Vibrate to indicate an action
     */
    void vibrateAction();
}

File: app/src/main/java/ca/warp7/android/scouting/abstraction/AbstractActionVibrator.kt
package ca.warp7.android.scouting.abstraction

/**
 * @since v0.4.2
 */

interface AbstractActionVibrator {

    /**
     * Vibrate to indicate scouting start
     */
    fun vibrateStart()

    /**
     * Vibrate to indicate an action
     */
    fun vibrateAction()
}
